RHEL7-STIG

```diff
@@ -0,0 +1 @@
+.
```

Comment:
didn't we already do this in https://github.com/MindPointGroup/RHEL7-STIG/commit/19af3fd6b8b67e53e948df0245058ab4413ef06b ?  We don't actually need it because we can use "{{ playbook_dir }}" to reference the role in the current directory.

Reply:
That is my fault I removed it in my testing update not realizing it was still used by local.yml